packages/firestore/src/remote/persistent_stream.ts

```diff
@@ -68,22 +85,16 @@ enum PersistentStreamState {
   /**
    * The stream encountered an error. The next start attempt will back off.
    * While in this state isStarted() will return false.
-   *
    */
   Error,
 
   /**
    * An in-between state after an error where the stream is waiting before
-   * re-starting. After
-   * waiting is complete, the stream will try to open. While in this
-   * state isStarted() will return YES but isOpen will return false.
+   * re-starting. After waiting is complete, the stream will try to open.
+   * While in this state isStarted() will return true but isOpen will return
```

Comment:
Nit: `While in this state, isStarted() will return true but isOpen() will return..`

I had to do a double take as I was reading this.